modules/appeals_api/app/models/appeals_api/non_veteran_claimant.rb

```diff
@@ -0,0 +1,155 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  class NonVeteranClaimant
+    def initialize(auth_headers:, form_data:)
+      @auth_headers = auth_headers
+      @form_data = form_data
+    end
+
+    def first_name
+      auth_headers['X-VA-Claimant-First-Name']
+    end
+
+    def middle_initial
+      auth_headers['X-VA-Claimant-Middle-Initial'] || ''
+    end
+
+    def last_name
+      auth_headers['X-VA-Claimant-Last-Name']
+    end
+
+    def ssn
+      auth_headers['X-VA-Claimant-SSN'] || ''
+    end
+
+    def birth_date_string
+      auth_headers['X-VA-Claimant-Birth-Date'] || ''
+    end
+
+    def birth_month
+      birth_date&.strftime('%m') || ''
+    end
+
+    def birth_day
+      birth_date&.strftime('%d') || ''
+    end
+
+    def birth_year
+      birth_date&.strftime('%Y') || ''
+    end
+
+    def full_name
+      "#{first_name} #{middle_initial} #{last_name}".squeeze(' ').strip
+    end
+
+    def number_and_street
+      address_combined || ''
+    end
+
+    def city
+      form_data['address']['city'] || ''
+    end
+
+    def state_code
+      form_data['address']['stateCode'] || ''
+    end
+
+    def country_code
+      form_data['address']['countryCodeISO2'] || 'US'
+    end
+
+    def zip_code_5
+      form_data['address']['zipCode5'] || '00000'
+    end
+
+    def email
+      form_data['email'].to_s.strip
+    end
+
+    def phone_data
+      form_data['phone']
+    end
+
+    def phone_formatted
+      AppealsApi::HigherLevelReview::Phone.new phone_data
+    end
+
+    def phone_string
+      phone_formatted.to_s
+    end
+
+    def area_code
+      phone_data&.dig('areaCode') || ''
+    end
+
+    def phone_prefix
+      return unless domestic_country_code_present?
```

Comment:
So here you return nil but if the phone is not domestic but also not filled out, you return a blank string. Let's at least keep this consistent with either returning nil or a blank string but not both.